### ISSUE TITLE
Why no get() method in RequestParams?

### ISSUE THREAD

**Issue by rterek** [loopj/android-async-http#1136]
It would be nice to fetch the value for a certain key but I see no way to do this. It's going to force me to store things in a map first and then convert to RequestParams, a tad annoying. Can this be added please?

**Comment by multivoltage** [comment]
I agree with you. Every time I need to "debug" my params I have to use hashMap and pass to client a new RequstParams(myHashMap)
